specs/Standards/Tokens/NonFungibleToken/Core.md

```diff
@@ -167,7 +167,7 @@ function nft_resolve_transfer(
   owner_id: string,
   receiver_id: string,
   token_id: string,
-  approved_account_ids: null|Record<string, number>,
+  approvals: null|Record<string, number>,
```

Comment:
While this [matches near-sdk-rs implementation](https://github.com/near/near-sdk-rs/blob/a903f8c44a7be363d960838d92afdb22d1ce8b87/near-contract-standards/src/non_fungible_token/core/resolver.rs#L29-L39), I scanned mainnet Indexer for Explorer database and there is no non-null `approvals` arguments (I have only scanned a subset of the recent data) while there are a bunch of non-empty `approved_account_ids`:

* Example of `approvals` usage that is set to null: https://explorer.near.org/transactions/A7ynVDwCb95KJtufhe1mygpsgGDRZT5nVX3vXK7Cg3Zg#FFnDDkoyFtTBseZgJyJZfeT84iDVivAYBQGQUKFQjKdc
* Example of `approved_account_ids`:
  * no set (neither `approved_account_ids` nor `approvals` is set): https://explorer.near.org/transactions/24RqapPjV7ky12zwW86kCb9xymbD2kix8zz7rpjpoThq#EzH1dXktEFyow15bejA8BSwyiCbuYNzyifpL8WchgVyx
  * set to an empty object (record): https://explorer.near.org/transactions/7PnKwGYtJheMrZCUKt6jRynfVeRiDrKFLZPseSAuMMrn#4XDwV6kMj9zi8NiXTiH2ZB6JLVPMGWreJpJdPAn1Jn7N
  * set to some meaningful value:
    * https://explorer.near.org/transactions/5wYkZRgegnmaYnFnvVaz6rsG8VvtgJz1TcnVTGVsLPVp#2yyw3vhuEGPsLUn2vi1HC2SyDNMdQL5zzSwjLwMRhsdY
    * https://explorer.near.org/transactions/9PfE5jBpHkUDSXh2CL9Y8xkKb6A8d13FcTvcKHfGjBo3#6iax6Qsxg5PFoy8wzdndkyKJs2S9BbzEQvEEV3nTyr4M

By the way, [near-sdk-js follows the NEP as expected](https://github.com/near/near-sdk-js/blob/2e0f9d3b78814742722db5384412972fd260a131/near-contract-standards/src/non_fungible_token/core/resolver.ts#L37).

@austinabell @ailisp @volovyks @near/wg-contract-standards I suggest we fix near-sdk-rs implementation instead of bending the NEP. Any objections? Better ideas?